<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeImageDialog</class>
 <widget class="QDialog" name="MakeImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make Image</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/res/images/app/48.png</normaloff>:/images/res/images/app/48.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Image from binary files</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="filesGroup">
         <property name="title">
          <string>Segment files and base addresses:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutFG"/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addFileBtn">
           <property name="text">
            <string>Add file</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/res/images/light/appbar.add.png</normaloff>:/images/res/images/light/appbar.add.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="entryPointGroup">
         <property name="title">
          <string>Image file and  entry point:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="ImageLineEdit" name="ouputFileLineEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="outputFileBtn">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/images/res/images/light/appbar.folder.open.png</normaloff>:/images/res/images/light/appbar.folder.open.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="HexLineEdit" name="entryPointLineEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="inputMask">
               <string>\0\xhhhhhhhh</string>
              </property>
              <property name="maxLength">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="makeImageBtn">
              <property name="text">
               <string>Make Image</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/images/res/images/light/appbar.settings.png</normaloff>:/images/res/images/light/appbar.settings.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Image from ELF file</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexLineEdit</class>
   <extends>QLineEdit</extends>
   <header>hexlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ImageLineEdit</class>
   <extends>QLineEdit</extends>
   <header>imagelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MakeImageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MakeImageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
